site/package-lock.json

```diff
@@ -5014,7 +5014,7 @@
         "is-installed-globally": "0.1.0",
         "lazy-ass": "1.6.0",
         "listr": "0.12.0",
-        "lodash": "4.17.11",
+        "lodash": "4.17.15",
```

Comment:
Interesting - something's patched cypress up to "4.17.15" lodash, was that you, or just from a npm install?

Reply:
That was just me trying to get the nom audit to pass after running npm install on the new package

Reply:
Nice - sounds a good approach.  Cypress 3.4.0 (latest) is still on 4.17.11, so makes sense for us to manually override